apps/angular/src/app/features/search-form/search-form.component.html

```diff
@@ -0,0 +1,9 @@
+<form class="search-form">
```

Comment:
If you specified `type=submit` for the button, you should use `ngSubmit` method for the form.
```suggestion
<form class="search-form" (ngSubmit)="onSubmit()">
```